indico/modules/events/abstracts/controllers/management.py

```diff
@@ -51,8 +52,12 @@ def _process(self):
             return WPManageAbstracts.render_template('management/disabled.html', self.event)
         else:
             abstracts_count = Abstract.query.with_parent(self.event).count()
+            custom_boa = None
+            if self.event.has_custom_boa:
+                custom_boa = BasicFileSchema().dump(self.event.custom_boa)
```

Comment:
Would this work the same using the previous schema but with [see below]?
```
FileSchema(only=('filename', 'size', 'uuid')).dump(self.event.custom_boa)
```

Reply:
Yep, that would work. But since this is something we likely need in more places in the future (whenever we use this field), a custom schema for it felt cleaner.